src/select-list-view.ts

```diff
@@ -136,8 +163,8 @@ module.exports = class SelectListView {
     }
 
     if (props.hasOwnProperty('query')) {
-      // Items will be recomputed as part of the change event handler, so we
-      // don't need to recompute them again at the end of this function.
+      /** Items will be recomputed as part of the change event handler, so we */
+      /** don't need to recompute them again at the end of this function. */
```

Comment:
Ditto with the comment style.

Reply:
Fixed in https://github.com/atom/atom-select-list/pull/28/commits/e5df058b6b014c6708eb6cb173ff25c8ef0cff4e